build_tools/github_actions/runner/config/setup.sh

```diff
@@ -14,19 +14,12 @@ set -xeEuo pipefail
 SCRIPT_DIR="$(dirname -- "$( readlink -f -- "$0"; )")";
 source "${SCRIPT_DIR}/functions.sh"
 
-RUNNER_TYPE="$(get_attribute github-runner-type)"
-# The CPU machines have 360GB of RAM
-TMPFS_SIZE=100g
-if [[ "${RUNNER_TYPE}" == gpu ]]; then
```

Comment:
Since we already have this special logic for GPU machines, I'm wondering if we can only add SSDs to GPU machines for now (and still use ramdisk on CPU machines)? So we don't need to put 16 SSDs for CPU machines until we have single logical volume

Reply:
Hmm yeah we could. It seems simpler to have them be the same. The single logical volume is possible right now, but it doesn't seem worth it because one is plenty of space and formatting them all took a while. I guess it's a question of how wasteful it is to have a bunch of unnecessary local SSDs. It looks like it increases the advertised price of the instance by about 20%. I'm assuming that the ratios are roughly the same for the actual cost. So ok, I think you're right that it's better to not do that. I'll rework.